model/src/main/scala/aqua/model/ValueModel.scala

```diff
@@ -15,10 +16,10 @@ object ValueModel {
   }
 }
 
-case class LiteralModel(value: String) extends ValueModel
+case class LiteralModel(value: String, `type`: Type) extends ValueModel
 
 object LiteralModel {
```

Comment:
Let's add `fromString` helper, that wraps a string into quotes and sets a proper type.